app/src/main/java/com/pileproject/drive/execution/ExecutionCondition.java

```diff
@@ -48,9 +48,6 @@ public SelectionResult(int index, boolean result) {
     }
 
     public ExecutionCondition(ArrayList<BlockBase> blocks) {
```

Comment:
Hmm, your choice was remove. That's ok, but you could change the signature from ArrayList to List.